shopify/order/copy-product-tag-to-order-tag-when-order-created/mesa.json

```diff
@@ -0,0 +1,89 @@
+{
+  "key": "copy_product_tag_to_order_tag_when_order_created",
+  "name": "Copy product tag to order tag when product is purchased",
+  "version": "1.0.0",
+  "description": "",
+  "video": "",
+  "readme": "",
+  "tags": [],
+  "source": "shopify_webhook",
+  "destination": "shopify_api",
+  "enabled": false,
```

Comment:
This is working pretty well :)

I didn't need to configure anything before enabling the workflow. Did you want to keep "enabled" as "false"?